.github/component_owners.yml

```diff
@@ -19,6 +19,8 @@ components:
     - thomaspoignant
   providers/jsonlogic-eval-provider:
     - justinabrahms
+  providers/unleash:
+    - liran2000
```

Comment:
@sighphyre @gastonfournier @ivarconr 

Would any of you like to be set as "component owners" here? This will add you as assignee/reviewers automatically if changes are made to files under this path in a PR.

We require only one name here, but I though I'd point it out.

Reply:
I'm very open to being added! I spend a lot of time on the Unleash SDKs so it'd be great to have an eye on this too!